core/src/test/java/io/questdb/test/griffin/wal/WalWriterFuzzTest.java

```diff
@@ -135,6 +135,15 @@ public void testWalAddRemoveCommitFuzzO3() throws Exception {
         runFuzz(rnd);
     }
 
+    @Test
+    public void testWalAddRemoveCommitMoveUncommittedSizeIssue() throws Exception {
+        Rnd rnd = fuzzer.generateRandom(LOG, 1512387976418L, 1713227441478L);
```

Comment:
The seeds quickly go out of date, practice shows very little point to have the tests with fixed seeds

Reply:
what would you suggest?

Reply:
I'd suggest removing the test as it is and if you think it's valuable to test this edge condition tweak the probabilities so that the chances of reproducing it on random seeds are high. Or write a non-fuzz test.

Reply:
Next week I'll add random column conversion in the fuzz gen procs and this seed will not reproduce the problem anymore.

Reply:
test removed